sdk/search/search-documents/samples/typescript/src/dataSourceConnections/deleteDataSourceConnectionByObject.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I feel like we should show both options for delete in one sample, and maybe even show the full CRUD flow in one -- create, fetch, update, delete.

Having so many little samples feels a little overkill